packages/@lwc/ssr-compiler/src/compile-js/generate-markup.ts

```diff
@@ -30,6 +31,7 @@ type RenderCallExpression = SimpleCallExpression & {
 
 const bGenerateMarkup = esTemplate`
     export async function* generateMarkup(tagName, props, attrs, slotted, parent, scopeToken) {
+        tagName = tagName ?? ${/*component tag name*/ is.literal}
```

Comment:
@nolanlawson I tried setting the default argument to the expression but it looks like it isn't set if a `null` value is passed to it, ie:

```js
yield* Ctor[SYMBOL__GENERATE_MARKUP](null, childProps, childAttrs);
```
The `tagName` in this case will show up as `null`.

I ended up keeping it as-is because passing `undefined` to the generator function seemed odd. 

We're also doing something similar with the `attrs` and `props` too.

Open to suggestions tho!